fv_be/app/models/category.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils.translation import gettext as _
+from django.core.exceptions import ValidationError
+
+# FirstVoices
+from .sites import BaseSiteContentModel
+
+
+class Category(BaseSiteContentModel):
+    """Model for Categories."""
+
+    # Fields
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    # i.e. Parent Categories can have children but those children cannot have more children
```

Comment:
I think it is good to have this comment here, but it is a bit confusing to me because it refers to having children when the field is actually for specifying a parent. Could we make the comment align? Something like "A category may have a parent, but grandparents are not allowed and this is enforced in the clean method"